app/views/users/edit.html.erb

```diff
@@ -1,171 +1,177 @@
-    method: :patch do |f| %>
-      <div>
+  <section>
+    <h2 id="user_timezone">Timezone</h2>
+    <%= form_with model: @user,
+      url: @is_own_settings ? my_settings_path : settings_user_path(@user),
+      method: :patch do |f| %>
+      <div class="form-group">
         <%= f.label :timezone, "Your timezone" %>
-        <%= f.select :timezone, 
+        <%= f.select :timezone,
             TZInfo::Timezone.all.map(&:identifier).sort,
-            include_blank: @user.timezone.blank? %>
+            include_blank: @user.timezone.blank?, class: "form-select" %>
         <small>This affects how your activity graph and other time-based features are displayed.</small>
       </div>
       <%= f.submit "Save Settings" %>
-  <% end %>
-</div>
+    <% end %>
+  </section>
 
-<div>
-  <h2 id="user_slack_status">Slack status</h2>
-  <% unless @can_enable_slack_status %>
-    <%= link_to "Re-authorize with Slack to give permission to update your status", slack_auth_path %>
-  <% end %>
-  <%= form_with model: @user,
-    url: @is_own_settings ? my_settings_path : settings_user_path(@user),
-    method: :patch do |f| %>
+  <hr>
+
+  <section>
+    <h2 id="user_slack_status">Slack status</h2>
+    <p>When you're hacking on a project, Hackatime can update your Slack status so you can show it off!</p>
+    <% unless @can_enable_slack_status %>
+      <%= link_to "Re-authorize with Slack to give permission to update your status", slack_auth_path %>
+    <% end %>
+    <%= form_with model: @user,
+      url: @is_own_settings ? my_settings_path : settings_user_path(@user),
+      method: :patch do |f| %>
       <fieldset>
         <label for="user_uses_slack_status">
           <%= f.check_box :uses_slack_status, id: "user_uses_slack_status" %>
           <%= f.label :uses_slack_status, "Update my Slack status with my current project" %>
         </label>
       </fieldset>
       <%= f.submit "Save Settings" %>
-  <% end %> 
-</div>
+    <% end %> 
+  </section>
 
-<div>
-  <h2 id="user_github_account">GitHub Account</h2>
-  <p>
-    This is used to show your active projects on the leaderboard & current hacking activity on the dashboard.
-  </p>
-  <% if @user.github_uid.present? %>
-    <p>Your GitHub account is linked. <%= link_to "@#{@user.github_username}", "https://github.com/#{@user.github_username}", target: "_blank" %></p>
-  <% else %>
-    <%= link_to "Link GitHub Account", github_auth_path, class: "button", data: { turbo: "false" } %>
-  <% end %>
-</div>
-
-<div>
-  <h2 id="user_slack_notifications">Slack notifications</h2>
-  <% if @enabled_sailors_logs.any? %>
-    <p>
-      You have notifications enabled for the following channels:
-    </p>
-    <ul>
-      <% @enabled_sailors_logs.each do |sl| %>
-        <li>
-          <%= render "shared/slack_channel_mention", channel_id: sl.slack_channel_id %>
-        </li>
-      <% end %>
-    </ul>
-  <% else %>
+    <section>
+    <h2 id="user_slack_notifications">Slack notifications</h2>
```

Comment:
I think something's a little funny with the indentation here– `section>h2` is nested.

Reply:
Fixed!